settings.gradle

```diff
@@ -23,6 +23,7 @@ rootProject.name = 'java-cfenv'
 
 include "java-cfenv"
 include "java-cfenv-boot"
+include "java-cfenv-tests-boot"
```

Comment:
I can't talk about details, but it should be checked this artifact does not get published with the releases. Based on `stage.sh` the task is `publish` (from 'maven-publish' plugin)

Reply:
yup, it's fixed, I used:

```
publishMavenJavaPublicationToMavenRepository.enabled = false
```

and tested that it did not want to publish anymore